README.md

```diff
@@ -107,36 +163,44 @@ Open the the folder where you've cloned the project
 Open the html file in your browser
 
 
+### Run tests
+
+To run tests, run the following command:
+
+No test required for this project
 
 ### Deployment
 
 You can deploy this project using:
 
 
-github pages
+using netlify
 
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
 
 👤 **Ayomide Sunday Emmanuel**
 
-- GitHub: https://github.com/Paulinelali/microverse-personal-portfolio
+- GitHub: https://github.com/Paulinelali/hello-microverse-project
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] kindly consider including all the partners who participated on this task to be than so as your readme look more professional